src/IIIFPresentation/API.Tests/Converters/Streaming/S3StoredJsonProcessorTests.cs

```diff
@@ -3,51 +3,56 @@
 using API.Converters.Streaming;
 using API.Helpers;
 using LateApexEarlySpeed.Xunit.Assertion.Json;
+using Microsoft.AspNetCore.Http;
 using Models.Database.General;
 
 namespace API.Tests.Converters.Streaming;
 
 public class S3StoredJsonProcessorTests
 {
+    private readonly IPathGenerator pathGenerator = new PathGenerator(new HttpContextAccessor()
```

Comment:
These 10 lines are duplicated in a lot of test classes - worth having something shared?